spec/system/task_spec.rb

```diff
@@ -1,36 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  include ApplicationHelper
+
   describe 'Task一覧' do
     context '正常系' do
+      let! (:project) { FactoryBot.create(:project) }
+      let! (:task) { FactoryBot.create(:task, project_id: project.id) }
```

Comment:
`FactoryBot.`の記載は設定ファイルに追記して省略して書くこともできます。
```suggestion
      let! (:task) { create(:task) }
```